.github/scripts/publish-recipes.sh

```diff
@@ -69,5 +65,5 @@ do
 
     echo "Publishing $RECIPE to $PUBLISH_REF"
     echo "- $PUBLISH_REF" >> $GITHUB_STEP_SUMMARY
-    $BICEP_EXECUTABLE publish $RECIPE --target "br:$PUBLISH_REF"
+    rad bicep publish --file $RECIPE --target "br:$PUBLISH_REF"
```

Comment:
using the bicep executable directly did not seem to work with GHCR.

Reply:
Yep this known behavior. The Bicep executable and our fork can only talk to ACR. We had to bring in the ability to talk to any OCI registry to the rad CLI directly.